docs/user-guide/node-selection/index.md

```diff
@@ -67,7 +67,7 @@ met, the pod will still continue to run on the node. In the future we plan to of
 `requiredDuringSchedulingRequiredDuringExecution` which will be just like `requiredDuringSchedulingIgnoredDuringExecution`
 except that it will evict pods from nodes that cease to satisfy the pods' node affinity requirements.
 
-Node affinity is currently expressed using an annotation on Pod. In v1.3 it will use a field, and we will
+Node affinity is currently expressed using an annotation on Pod. In v1.4 it will use a field, and we will
```

Comment:
Please change "In v1.4 it will use a field, and we will also introduce the second piece of the affinity mechanism, called pod affinity," to "After the release of 1.4, node affinity will use a field, and it will introduce an additional option for affinity called "pod affinity","

Three reasons for this: We won't have edit this again for the 1.5 release; we should avoid using the ambiguous pronoun "we" in our docs; and we should explicitly call out the difference between node and pod affinity.